recipes-core/sysvinit/sysvinit-inittab_%.bbappend

```diff
@@ -0,0 +1,7 @@
+do_install_append() {
+    sed -i '1 a \
+\
+# Disable serial console via envParams.SERIAL_CONSOLE_DISABLE\
+while [ -f "/tmp/config_serial_console_disable" ]; do sleep 10; done\
```

Comment:
Can we use if instead while?

if [ ! -f "/tmp/config_serial_console_disable" ]; then
   ' ${D}${base_bindir}/start_getty
fi

Reply:
Not sure if you are misreading this code -- it is injecting these specific lines into the `start_getty` script via `sed` (this is not executed during the build).

The `while` is necessary or else init will automatically restart this script (infinite loop).